apps/voting-disputable/app/src/components/VoteDetail/DisputableActionStatus.js

```diff
@@ -34,6 +36,9 @@ function DisputableActionStatus({ vote }) {
   const disputableStatus =
     vote.disputable && DISPUTABLE_VOTE_STATUSES.get(vote.disputable.status)
```

Comment:
Just curious, is likely to be `undefined`?

Reply:
Is not impossible. I think it could happen if the request fails for some reason